src/Lean/Server/InfoUtils.lean

```diff
@@ -108,7 +108,7 @@ def Info.stx : Info → Syntax
   | ofFieldInfo i          => i.stx
   | ofCompletionInfo i     => i.stx
   | ofCustomInfo i         => i.stx
-  | ofUserWidgetInfo i     => i.stx
+  | ofPanelWidgetInfo i    => i.stx
```

Comment:
FWIW, @david-christiansen mentioned that he had great success with using `ofCustomInfo`, which begs the question whether we want to migrate all other constructors to it /cc @mhuisi

Reply:
Ah, I was [using that](https://github.com/leanprover-community/ProofWidgets4/blob/28046b1c0d70de55bc3ef33abef635ac4dc9a5ab/ProofWidgets/Compat.lean#L159) in ProofWidgets to hack around things that this PR is changing. It has the added benefits of cutting some dependencies, i.e., `InfoTree.Types` would not need to know about widgets anymore. It may also mesh well with your plans to generalize info trees. However, I had undone this renaming now as it's more annoying than helpful, so I'd leave that as future work.

Reply:
Great, fine with me. @mhuisi mentioned that it might be a good idea to keep separate constructors for "standard" cases - which arguably widgets would be more so than the other constructors that are specific to the Lean language